src/providers/WalletProvider.tsx

```diff
@@ -1,9 +1,18 @@
 import { UseWalletProvider } from 'use-wallet';
+import { RPC_URLS } from '../constants/rpc';
 
 interface Props {
   children: React.ReactNode;
 }
 
 export default ({ children }: Props) => (
-  <UseWalletProvider>{children}</UseWalletProvider>
+  <UseWalletProvider
+    connectors={{
+      walletconnect: {
+        rpc: RPC_URLS,
```

Comment:
This is the correct configuration, but differs from what's defined in the [use-wallet documentation](https://github.com/aragon/use-wallet#connectors) -- will open a PR against the repo soon to resolve that.